allauth/socialaccount/providers/lemonldap/provider.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+from allauth.socialaccount.providers.base import ProviderAccount
+from allauth.socialaccount.providers.oauth2.provider import OAuth2Provider
+
+
+class LemonldapAccount(ProviderAccount):
```

Comment:
Nitpick, but I think it would be best to rename this (and other classes) to: `LemonLDAPAccount`.

Reply:
Sure, I just updated the PR